### DISCUSSION TITLE
Возможно ли что будет обновлена меню игры?

### DISCUSSION THREAD

**Discussion by gat3v** [Sokol-Games-GmbH/PixelCombats.PublicApi#141] [Q&A]
**Хочется увидеть что нибудь особенное в меню игры (изменение, красочность, особенность)**

**Comment by kkohno** [comment]
в каком именно меню?

**Comment by kkohno** [comment]
> **Хочется увидеть что нибудь особенное в меню игры (изменение, красочность, особенность)**

такое пойдет?))

![image](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/17728172/868d539e-67ac-4263-ba61-a4ce8565e978)
![image](https://github.com/Sokol-Games-GmbH/PixelCombats.PublicApi/assets/17728172/8fbadfae-8901-4e62-8915-2a47d2d859c7)

**Reply by gat3v** [reply]
вау, это уже круто)

**Reply by ZhenyaMartynov** [reply]
Стало хуже...
Будет же переделыватся?

**Comment by ZhenyaMartynov** [comment]
![Uploading pixelcut-export-1697889160312.png…]()
